### PR TITLE
Update sudoers and Makefile

### LINKED ISSUE #3448: [security] domain join: logs contain clear-text credentials
When you declare a new Active Directory Domain in PacketFence's config and try to join it, on CentOS 7, you can see clear-text credentials used in `/var/log/secure`.

### PR DESCRIPTION
**Pull request by nqb**
# Description
- Don't log `ip netns exec` commands to fix #3448 
- `make sudo` doing same things as packaging

# Impacts
- use of sudo by pf user
- `make devel`

# Issue
fixes #3448

# Delete branch after merge
YES

### CODE REVIEW

**Comment by nqb** [comment]
@fdurand, @julsemaan 

Do you think we need to add a section in UPGRADE.asciidoc for `sudoers` file because users will get `rpmnew` or `dist` files ?

**Comment by julsemaan** [comment]
Its the user's responsibility to go over .rpmnew files usually but this is only documented for files in /usr/local/pf

Since that one isn't there, I'd say we could add a note in the upgrade

File: UPGRADE.asciidoc [hunk 1]
```diff
@@ -1776,6 +1776,13 @@ The realms are now multi-tenant aware, in order to upgrade your configuration to
 
   /usr/local/pf/addons/upgrade/to-8.2-realm-conf.pl
 
+
+Upgrade sudoers file (All linux distributions)
```
Comment (extrafu) [inline UPGRADE.asciidoc]: all Linux

File: UPGRADE.asciidoc [hunk 2]
```diff
@@ -1776,6 +1776,13 @@ The realms are now multi-tenant aware, in order to upgrade your configuration to
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`sudo` configuration will now hide commands starting by `/sbin/ip netns exec` in syslog file. During upgrade of PacketFence package, package manager of your distribution will inform you that `/etc/sudoers.d/packetfence` has changed. You will need to merge current and new `sudoers` file.
```
Comment (extrafu) [inline UPGRADE.asciidoc]: starting with
During the PacketFence upgrade, the package manager
to merge the current

File: UPGRADE.asciidoc (hunk 1 above)
Comment (nqb) [reply UPGRADE.asciidoc]: @extrafu : in that case, I will update many lines in that file (see 'Database schema update' titles).

**Comment by nqb** [comment]
@extrafu: you can review

**Comment by extrafu** [comment]
linux -> Linux
During the Packetfence upgrade -> During PacketFence upgrade

**Comment by nqb** [comment]
> During the Packetfence upgrade -> During PacketFence upgrade

@extrafu: already done.

**Comment by nqb** [comment]
@extrafu: rebase done, you can merge.

**Pull request merged**